README.md

```diff
@@ -1 +1,101 @@
-# shin_capstone
\ No newline at end of file
+# shin_capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+
+> This is a module 1: week 5 project in Microverse full stack course.
+
+
+## Built With
+
+- html
+- css
+- Javascript
+
+
+## Live Demo
+
+[**See Project 🚀**](https://strangeal.github.io/shin_capstone/)
+
+
+## Video Demonstration
+
+[** Watch 😄 **](https://drive.google.com/file/d/1bm7f6Amoa8ATpR5l63DTtfm6wE2HDOCy/view?usp=sharing)
+
+## Screenshots 
+
+![Desktop Design](./img/desktopview.png)
+
+## Get Started
+
+To get the content of this project locally you need to run the following commands in your terminal.
+
+### Prerequisites
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone git@github.com:Strangeal/Portfolio.git
+```
+3. To access cloned directory run:
+```bash
+cd MyPortfolio
+```
+3. Open it with the live server
+
+### Install
+> To install linters and other project's dependencies run:
+```bash
+npm install
+```
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+## Author
+
+👤 **Justice Bajeri**
+
+- GitHub: [@strangeal](https://github.com/Strangeal)
+- Twitter: [@str_angeal](https://twitter.com/Str_angeal)
+- LinkedIn: [Justice Bajeri](https://www.linkedin.com/in/justice-bajeri-0b7211243/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Inspiration
+- etc
+
```

Comment:
- [ ] In the project requirements, you are expected to give credit and  Acknowledgements to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
Please add the credit here in the acknowledgments section 👍